wallet/src/Pos/Wallet/Web/Api.hs

```diff
@@ -503,6 +506,21 @@ data WBackupApiRecord route = WBackupApiRecord
   }
   deriving (Generic)
 
+-- ~~~~~~~~~~
+--   /system
+-- ~~~~~~~~~~
+
+type WSystemApi = "system" :> ToServant (WSystemApiRecord AsApi)
+
+data WSystemApiRecord route = WSystemApiRecord
+  {
+    _requestShutdown :: route
+    :- "shutdown"
+    :> Summary "Import full information about wallet from a given file."
```

Comment:
ahaha, it seems you copied this from import endpoint and forgot to rewrite, :lalka:

Reply:
Lol, nice.

Reply:
ahaha loh ppz mda :kaby: